### PR TITLE
Dynamic config resource cleanup

### PR DESCRIPTION
**Pull request by AvaniMakwana**
Authored-by: AvaniMakwana

Description
Clean up Dynamic config module test resources

Motivation and Context
This will remove redundant test resources from elide dynamic config

How Has This Been Tested?
Unit tests included / updated

Screenshots (if appropriate):
License
I confirm that this contribution is made under an Apache 2.0 license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: elide-contrib/elide-dynamic-config-helpers/src/test/java/com/yahoo/elide/contrib/dynamicconfighelpers/parser/handlebars/HandlebarsHydratorTest.java [hunk 1]
```diff
@@ -154,6 +161,7 @@
             + "        tableSource=\"\"\n"
             + "    )\n"
             + "    \n"
+//            + "    \n"
```
Comment (moizarafat) [inline elide-contrib/elide-dynamic-config-helpers/src/test/java/com/yahoo/elide/contrib/dynamicconfighelpers/parser/handlebars/HandlebarsHydratorTest.java:164]: Please remove commented code.

File: elide-contrib/elide-dynamic-config-helpers/src/test/java/com/yahoo/elide/contrib/dynamicconfighelpers/parser/handlebars/HandlebarsHydratorTest.java [hunk 2]
```diff
@@ -210,12 +218,12 @@
             + "        tags={\"PUBLIC\"}\n"
             + "    )\n"
             + "    \n"
+//            + "    \n"
```
Comment (moizarafat) [inline elide-contrib/elide-dynamic-config-helpers/src/test/java/com/yahoo/elide/contrib/dynamicconfighelpers/parser/handlebars/HandlebarsHydratorTest.java:221]: Please remove commented code.

**Review by moizarafat — CHANGES_REQUESTED** [review]

File: elide-contrib/elide-dynamic-config-helpers/src/test/resources/validator/valid/db/variables.hjson [hunk 3]
```diff
@@ -1,4 +1,4 @@
 {
-   mysql_dialect: com.yahoo.elide.datastores.aggregation.queryengines.sql.dialects.impl.HiveDialect
-   db2_dialect: com.yahoo.elide.datastores.aggregation.queryengines.sql.dialects.impl.PrestoDBDialect
+   mysql_dialect: org.hibernate.dialect.MySQL5Dialect
```
Comment (aklish) [inline elide-contrib/elide-dynamic-config-helpers/src/test/resources/validator/valid/db/variables.hjson:2]: This is incorrect.  These dialects should be elide dialects (like before).

File: elide-contrib/elide-dynamic-config-helpers/src/test/resources/validator/valid/db/sql/single_db.hjson [hunk 4]
```diff
@@ -6,7 +6,7 @@
       url: jdbc:oracle:thin:@localhost:1521:testdb
       driver: oracle.jdbc.driver.OracleDriver
       user: guestoracle
-      dialect: com.yahoo.elide.datastores.aggregation.queryengines.sql.dialects.impl.H2Dialect
+      dialect: org.hibernate.dialect.OracleDialect
```
Comment (aklish) [inline elide-contrib/elide-dynamic-config-helpers/src/test/resources/validator/valid/db/sql/single_db.hjson:9]: Incorrect, change back to what was there before.

File: elide-contrib/elide-dynamic-config-helpers/src/test/resources/validator/valid/db/sql/multiple_db.hjson [hunk 5]
```diff
@@ -6,7 +6,7 @@
       url: jdbc:db2:localhost:50000/testdb
       driver: COM.ibm.db2.jdbc.net.DB2Driver
       user: guestdb2
-      dialect: <% db2_dialect %>
```
Comment (aklish) [inline elide-contrib/elide-dynamic-config-helpers/src/test/resources/validator/valid/db/sql/multiple_db.hjson:9]: Incorrect, revert to the older code.

File: elide-contrib/elide-dynamic-config-helpers/src/test/resources/validator/invalid_schema/db_invalid.hjson [hunk 6]
```diff
@@ -6,7 +6,7 @@
       url: jdbc:db2:localhost:50000/testdb
       driver: COM.ibm.db2.jdbc.net.DB2Driver
       user: guestdb2
-      dialect: com.yahoo.elide.datastores.aggregation.queryengines.sql.dialects.impl.PrestoDBDialect
+      dialect: org.hibernate.dialect.DB2Dialect
```
Comment (aklish) [inline elide-contrib/elide-dynamic-config-helpers/src/test/resources/validator/invalid_schema/db_invalid.hjson:9]: Incorrect - revert to prior version.

**Review by aklish — CHANGES_REQUESTED** [review]

**Pull request merged**